services/viva-ms/lambdas/syncExpiryTime.js

```diff
@@ -49,6 +50,10 @@ function getExpiryHoursByStatusType(statusType) {
   const statusHourMap = {
     'active:ongoing': DELETE_VIVA_CASE_AFTER_72_HOURS,
     'active:submitted:viva': DELETE_VIVA_CASE_AFTER_45_DAYS,
+    'closed:approved:viva': DELETE_VIVA_CASE_AFTER_6_MONTH,
+    'closed:partiallyApproved:viva': DELETE_VIVA_CASE_AFTER_6_MONTH,
+    'closed:rejected:viva': DELETE_VIVA_CASE_AFTER_6_MONTH,
+    'closed:completionRejected:viva': DELETE_VIVA_CASE_AFTER_6_MONTH,
```

Comment:
Checking the case statuses list there is also a status just called `closed`; should that one be considered as well?